pages/common/vault.md

```diff
@@ -0,0 +1,27 @@
+# vault
+
+> HashiCorp Vault, a tool for managing secrets.
```

Comment:
Can we remove "HashiCorp Vault" ? It does not add any value to the explanation.

Reply:
Would it be better if I changed it like this?

    > A CLI to interact with HashiCorp Vault

The 'vault' tool does nothing on it's own, you only use it to interact with a Vault server (locally or remotely).